Gemfile

```diff
@@ -6,11 +6,12 @@ gem 'rails', '4.2.11.3'
 gem 'acts-as-taggable-on', '~> 5.0'
 gem 'carrierwave'
 gem 'carrierwave-ftp', require: 'carrierwave/storage/sftp'
-gem 'compass-rails', git: 'https://github.com/Compass/compass-rails'
+gem 'compass-rails'
```

Comment:
I had to update this to get the new Bootstrap working, let me know @despo if you think it'll be an issue.

Reply:
No idea! I thought it had to do with foundation but that does seem to be the case (related commit: 
https://github.com/codebar/planner/commit/7b57dc5fc9d6c7baef5ebb3f58b305d14be9a94b)

We should probably try to push this fix on its own to staging and see if it causes any issues with deployment. If not we should be fine.

Reply:
Sounds good, I'll do a separate PR for this to test staging 👍

Reply:
Separate PR here https://github.com/codebar/planner/pull/1421